app/sections/blogs/index.tsx

```diff
@@ -107,10 +107,15 @@ const Blogs = forwardRef<HTMLElement, BlogProps>((props, ref) => {
     }
 
     let res = loaderData?.blog?.articles.nodes;
-
+    const truncateText = (content: any, maxLength: number) => {
```

Comment:
Looks like Tailwind [Line Clamp](https://tailwindcss.com/docs/line-clamp) utils can resolve this instead of using a custom `truncateText` function, can you give it a try?

Reply:
So we will have to set max height for content description

Reply:
it look like this
<div className='whitespace-normal truncate max-h-40' dangerouslySetInnerHTML={{ __html: idx.contentHtml }}></div>

Reply:
Can you remove all the class names and try to use only `line-clamp-*` ?

Btw, is it correct to render the `contentHtml` here 🤔? I think we should use [`article.excerpt`](https://shopify.dev/docs/api/storefront/2024-04/objects/Article#field-article-excerpt) for a short article summary instead of render the full article content.

Reply:
I have fixed it, please check again